roles/dev/tasks/main.yml

```diff
@@ -0,0 +1,107 @@
+---
+# Devlopment specific tasks
+
+- name: Transport Layer Security (setup_ssl)
+  include_tasks: setup_ssl.yml
+  when: setup_ssl
+
+# Maildev
+- name: Install PHP PEAR modules
+  become: true
+  pear: name={{item}} state=present
+  with_items: { Auth_SASL, mail, mail_mime, Net_SMTP }
+
+- name: Install Maildev for catching emails while developing
+  become: true
+  npm: name=maildev version=1.0.0-rc3 state=present global=yes
+  notify: restart maildev
+
+- name: Setup Maildev to start on reboot
+  become: true
+  file: state=link src={{ dir.script }}/init_maildev.sh path=/etc/init.d/maildev mode='755'
+
+- name: Create Maildev startup symlink
+  # TODO insserv Script maildev is broken incomplete LSB comment.
+  become: true
+  file: state=link src=/etc/init.d/maildev path=/etc/rc3.d/S99maildev
+
+# MariaDB
+- name: Save db credentials locally
+  # See https://dev.mysql.com/doc/refman/5.7/en/option-files.html
+  # TODO use vault instead
+  copy:
+    dest: ~/.my.cnf
+    content: |
+      [client]
+      user={{db.username}}
+      password={{db.password}}
+
+# PHP debug
+# https://github.com/Hitchwiki/hitchwiki/blob/master/configs/mediawiki.php#L59-L63
+- name: Turn on PHP error reporting
+  become: true
+  lineinfile:
+    path: /etc/php/7.0/apache2/php.ini
+    regexp: '^error_reporting = .*'
+    line: 'error_reporting = E_ALL'
+  when: debug
+  notify: restart apache
```

Comment:
Let's have PHP error reporting _off_ by default (it should already be?) so that we can turn it _on_ then on _runtime only_ (at mediawiki.php). Kinda just more secure practise in that way. :-)